homeassistant/components/xiaomi_miio/sensor.py

```diff
@@ -147,31 +169,149 @@ class XiaomiMiioSensorDescription(SensorEntityDescription):
         icon="mdi:fast-forward",
         state_class=STATE_CLASS_MEASUREMENT,
     ),
+    ATTR_MOTOR2_SPEED: XiaomiMiioSensorDescription(
+        key=ATTR_MOTOR2_SPEED,
+        name="Second Motor Speed",
+        native_unit_of_measurement="rpm",
+        icon="mdi:fast-forward",
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
     ATTR_ILLUMINANCE: XiaomiMiioSensorDescription(
         key=ATTR_ILLUMINANCE,
         name="Illuminance",
         native_unit_of_measurement=UNIT_LUMEN,
         device_class=DEVICE_CLASS_ILLUMINANCE,
         state_class=STATE_CLASS_MEASUREMENT,
     ),
+    ATTR_ILLUMINANCE_LUX: XiaomiMiioSensorDescription(
+        key=ATTR_ILLUMINANCE,
+        name="Illuminance",
+        native_unit_of_measurement=LIGHT_LUX,
+        device_class=DEVICE_CLASS_ILLUMINANCE,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
     ATTR_AIR_QUALITY: XiaomiMiioSensorDescription(
         key=ATTR_AIR_QUALITY,
         native_unit_of_measurement="AQI",
         icon="mdi:cloud",
         state_class=STATE_CLASS_MEASUREMENT,
     ),
+    ATTR_PM25: XiaomiMiioSensorDescription(
+        key=ATTR_AQI,
+        name="PM2.5",
+        native_unit_of_measurement=CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
+        icon="mdi:blur",
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    ATTR_FILTER_LIFE_REMAINING: XiaomiMiioSensorDescription(
+        key=ATTR_FILTER_LIFE_REMAINING,
+        name="Filter Life Remaining",
+        native_unit_of_measurement=PERCENTAGE,
+        icon="mdi:air-filter",
+        state_class=STATE_CLASS_MEASUREMENT,
+        attributes=(
+            "filter_hours_used",
+            "filter_rfid_product_id",
```

Comment:
This sounds like it's static info. We only want to have dynamic info related to the entity state in the state attributes. There are only some exceptions like attribution or if the info is essential for automations.

Reply:
This `filter_rfid_product_id` looks like this : 0:0:30:33. It could change if you use a different filter product.